addon/-private/utils/send-action.js

```diff
@@ -0,0 +1,10 @@
+import { SUPPORTS_CLOSURE_ACTIONS } from 'ember-compatibility-helpers';
+
+export default function(context, action, ...params) {
+  if (SUPPORTS_CLOSURE_ACTIONS) {
+    if (typeof context[action] === 'function') {
+      context[action](...params);
```

Comment:
Without the return statement here, couldn't this now trigger the action twice if it's a closure action?

Reply:
Yes. The proper factoring is:

```js
export default function(context, action, ...params) {
  if (SUPPORTS_CLOSURE_ACTIONS && typeof context[action] === 'function') {
    context[action](...params);
  } else {
    context.sendAction(action, ...params);
  }
}
```

The goal of not returning from the action is that we don't want to presume the return value from closure actions are available, since they won't be when calling `sendAction`.

Reply:
great catch, made the change